### DISCUSSION TITLE
Orderbook and Volume Data

### DISCUSSION THREAD

**Discussion by maxschulmeister** [SockTrader/SockTrader#81] [Ideas]
Since we are connected through ws to the exchange, this is a perfect opportunity to get orderbook and volume data and do something with it. e.g. use in strategies. This data is not available on Tradingview.

**Comment by cwouter** [comment]
Volume is already extracted from the candle stream (https://github.com/SockTrader/SockTrader/blob/socktrader-v2/src/exchanges/binance/mapper.ts#L12). I agree with orderbook data, this one was available in SockTrader v1. For v2 i'll consider this a low prio feature (for now) compared to the other stuff that is on our roadmap.
